wgsl/index.bs

```diff
@@ -4180,6 +4180,242 @@ TODO: deduplicate these tables
 
 ## Texture built-in functions ## {#texture-builtin-functions}
 
+### `textureSample` ### {#texturesample}
+
+Samples a texture.
+
+```rust
+textureSample(texture : texture_sampled_1d<type>,         sampler : sampler, coords : f32                                             ) -> <type>
```

Comment:
these should all return `vec4<f32>`. As indicated in #1162, we can't portably sample non-float textures

Reply:
Done